lua/hop/hint.lua

```diff
@@ -203,6 +203,7 @@ function M.set_hint_extmarks(hl_ns, per_line_hints)
       end
     end
   end
+  vim.cmd('redraw')
```

Comment:
Isn’t there a Lua API for this? And also, I think this shouldn’t be done in this function. Imagine that we want to, for whatever reason at some point, set the marks and do a bunch of other visual things: we will want to `redraw` afterwards.

Reply:
I agree. Maybe an optional `redraw` that defaults to `false`?

I originally had the `redraw` call in the parent function, but then I realized that it would only ever be called right after the hints would be updated.

Reply:
Ah, re:the other question—I handn't found a API/Lua based function for this. There are a few API functions that deal with redraws (`nvim_set_decoration_provider`, `nvim__buf_redraw_range`), but nothing I could find that would initiate a screen redraw.

Reply:
Since I believe the redraw comments are *only* necessary because we do "our own keyhandling loop" now, I've added them in `hint_with`. Please tell me if that doesn't make sense for you and I'll do it differently :)

Reply:
That’s perfect, thanks a lot!